### PR TITLE
Create Contributing.md for team principles

### PR DESCRIPTION
**Pull request by hdrury1**
Adding team working principles ADR

### CODE REVIEW

File: docs/adr/0004-principles.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Decision
+
+* We will prioritise reviewing Pull Requests (PRs) over beginning new issues.
+* Where possible, issues will include a user story and product requirements/acceptance criteria as well as links to relevant designs.
+* Where possible, issues will be grouped together according to the product feature they relate to.
+* After two days' effort, an assignee will add a comment to an issue and/or consider raising new issues to clearly indicate progress.
+* When feedback is required, an assignee will open a PR with the 'WIP' prefix. See [this ADR](https://github.com/elifesciences/elife-xpub/blob/develop/docs/adr/0001-pull-request-workflow.md).
+* As we facilitate deployment on a per-branch basis, once a PR is merged it is considered 'Done'.
+* Further end-to-end testing will be raised in a new issue with the 'e2e test' label.
```
Comment (tamlyn) [inline docs/adr/0004-principles.md]: If a feature can naturally be tested as part of an existing end-to-end test flow, then I think it should usually be done in the same issue.

File: docs/adr/0004-principles.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* An issue will never be re-opened.
```
Comment (tamlyn) [inline docs/adr/0004-principles.md]: Unless closed in error?

**Comment by hdrury1** [comment]
My reasoning behind capturing these principles in an ADR is that they affect contruction techniques (in the same way we have documented the use of PRs).

**Comment by hdrury1** [comment]
Okay folks. Attempt number 2 deletes the ADR and introduces a Contributing.md to the root.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Contributing to xPub
+
+Date Last Updated: 15.06.18
```
Comment (tamlyn) [inline CONTRIBUTING.md]: I'll let git be the judge of that!

**Pull request merged**